components/Common/Button/index.module.scss

```diff
@@ -0,0 +1,106 @@
+@import 'styles/new/index';
+
+.button {
+  @apply px-[18px]
+   py-[10px]
+   font-semibold
+   relative;
+
+  &.primary {
+    @apply bg-green-600
+     text-white
+     rounded
+     border
+     border-green-600
+     shadow-sm;
+
+    &:hover {
+      @apply bg-green-700
+       border-green-700;
+    }
+
+    &:focus {
+      @apply bg-green-700
+       border-green-700;
+    }
+
+    &:disabled {
+      @apply bg-green-600
+       opacity-50;
+    }
+  }
+
+  &.secondary {
+    @apply text-neutral-800
+     rounded-lg
+     disabled:opacity-50
+     dark:text-neutral-200/100;
+
+    &:hover {
+      @apply text-neutral-800
+       bg-neutral-100
+       dark:bg-neutral-900
+       dark:text-neutral-200;
+    }
+
+    &:disabled {
+      @apply bg-neutral-100/0;
+    }
+
+    &:focus {
+      @apply text-neutral-800
+       bg-neutral-100
+       dark:bg-neutral-900
+       dark:text-neutral-200;
+    }
+  }
+
+  &.special {
+    @apply bg-green-600/10
+     text-white
+     shadow-sm
+     rounded-lg
+     border
+     border-green-600/30;
+
+    &::before {
+      @apply absolute
+       h-full
+       w-full
+       top-0
+       right-0;
+      @include glow-backdrop(8em, #5fa04e);
+      content: '';
+    }
+
+    &::after {
+      background: var(
+        --gradient-glow,
+        linear-gradient(
+          90deg,
+          rgba(65, 126, 56, 0) 0%,
+          #417e38 50%,
+          rgba(65, 126, 56, 0) 100%
+        )
+      );
+      content: '';
+      height: 1px;
```

Comment:
Note, that you should still use tailwind variables for this; such as `height-[1px]`

We shouldn't do any manual CSS here if possible